web/styles/zulip.css

```diff
@@ -550,7 +550,7 @@ body.has-overlay-scrollbar {
     }
 
     .column-right .right-sidebar {
-        padding-left: 10px;
+        padding-left: 5px;
```

Comment:
Similarly does this 5px need to be the same as some other variable such that it could be a variable?

Reply:
This is the left padding here, not shared with anything:

<img width="274" alt="image" src="https://github.com/user-attachments/assets/6935d0e6-dd0e-4966-9261-96555302118a">